<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>LRig_Form</class>
 <widget class="QWidget" name="LRig_Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>301</width>
    <height>729</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Light Rig</string>
  </property>
  <widget class="QLabel" name="lblHeader">
   <property name="geometry">
    <rect>
     <x>-70</x>
     <y>10</y>
     <width>451</width>
     <height>61</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Roboto</family>
     <pointsize>28</pointsize>
    </font>
   </property>
   <property name="lineWidth">
    <number>1</number>
   </property>
   <property name="text">
    <string>Light Rig</string>
   </property>
   <property name="textFormat">
    <enum>Qt::AutoText</enum>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="lblOUT">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>690</y>
     <width>261</width>
     <height>31</height>
    </rect>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="text">
    <string>...</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QListWidget" name="nodeListV">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>80</y>
     <width>261</width>
     <height>81</height>
    </rect>
   </property>
  </widget>
  <widget class="QLCDNumber" name="lcdNumber">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>170</y>
     <width>81</width>
     <height>23</height>
    </rect>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <property name="lineWidth">
    <number>0</number>
   </property>
   <property name="digitCount">
    <number>8</number>
   </property>
   <property name="value" stdset="0">
    <double>0.000000000000000</double>
   </property>
   <property name="intValue" stdset="0">
    <number>0</number>
   </property>
  </widget>
  <widget class="QRadioButton" name="node_errorOp">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>170</y>
     <width>191</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Abort on Non-Arnold Light Nodes.</string>
   </property>
  </widget>
  <widget class="QPushButton" name="bttn_deSelNodes">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>200</y>
     <width>121</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>De-Select All</string>
   </property>
  </widget>
  <widget class="QPushButton" name="bttn_nodes">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>200</y>
     <width>121</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>Select Nodes</string>
   </property>
  </widget>
  <widget class="QTabWidget" name="tabWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>250</y>
     <width>261</width>
     <height>441</height>
    </rect>
   </property>
   <property name="tabPosition">
    <enum>QTabWidget::North</enum>
   </property>
   <property name="tabShape">
    <enum>QTabWidget::Rounded</enum>
   </property>
   <property name="currentIndex">
    <number>0</number>
   </property>
   <widget class="QWidget" name="tab">
    <attribute name="title">
     <string>   Colour/Exposure    </string>
    </attribute>
    <widget class="Line" name="line">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>250</y>
       <width>201</width>
       <height>16</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>190</y>
       <width>31</width>
       <height>16</height>
      </rect>
     </property>
     <property name="layoutDirection">
      <enum>Qt::RightToLeft</enum>
     </property>
     <property name="text">
      <string>Green:</string>
     </property>
    </widget>
    <widget class="QLabel" name="r_out">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>70</y>
       <width>41</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>R:</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
     </property>
    </widget>
    <widget class="QSlider" name="Cd_Slider_B">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>220</y>
       <width>160</width>
       <height>22</height>
      </rect>
     </property>
     <property name="maximum">
      <number>100</number>
     </property>
     <property name="value">
      <number>50</number>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>220</y>
       <width>31</width>
       <height>16</height>
      </rect>
     </property>
     <property name="layoutDirection">
      <enum>Qt::RightToLeft</enum>
     </property>
     <property name="text">
      <string>Blue:</string>
     </property>
    </widget>
    <widget class="QLabel" name="b_out">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>130</y>
       <width>41</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>B:</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="lbl_LGTC_PRV">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>211</width>
       <height>271</height>
      </rect>
     </property>
     <property name="text">
      <string>Colour</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QPushButton" name="bttn_multiply">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>370</y>
       <width>111</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Multiply</string>
     </property>
    </widget>
    <widget class="QLabel" name="g_out">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>100</y>
       <width>41</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>G:</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="Minlbl">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>260</y>
       <width>61</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>MS Shell Dlg 2</family>
       <pointsize>8</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Intensity:</string>
     </property>
    </widget>
    <widget class="QPushButton" name="bttn_setExposure">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>330</y>
       <width>231</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Set Values</string>
     </property>
    </widget>
    <widget class="QSlider" name="Cd_Slider_G">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>190</y>
       <width>160</width>
       <height>22</height>
      </rect>
     </property>
     <property name="maximum">
      <number>100</number>
     </property>
     <property name="value">
      <number>50</number>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>160</y>
       <width>21</width>
       <height>16</height>
      </rect>
     </property>
     <property name="layoutDirection">
      <enum>Qt::RightToLeft</enum>
     </property>
     <property name="text">
      <string>Red:</string>
     </property>
    </widget>
    <widget class="QDoubleSpinBox" name="multSpinBox">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>370</y>
       <width>101</width>
       <height>31</height>
      </rect>
     </property>
     <property name="decimals">
      <number>2</number>
     </property>
    </widget>
    <widget class="QLabel" name="lbl_LGTC_PRV_M">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>211</width>
       <height>271</height>
      </rect>
     </property>
     <property name="text">
      <string>MASK</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QDoubleSpinBox" name="MaxdoubleSpinBox">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>290</y>
       <width>81</width>
       <height>31</height>
      </rect>
     </property>
     <property name="wrapping">
      <bool>false</bool>
     </property>
     <property name="suffix">
      <string/>
     </property>
     <property name="decimals">
      <number>2</number>
     </property>
     <property name="maximum">
      <double>1000000.000000000000000</double>
     </property>
    </widget>
    <widget class="QSlider" name="Cd_Slider_R">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>160</y>
       <width>160</width>
       <height>20</height>
      </rect>
     </property>
     <property name="maximum">
      <number>100</number>
     </property>
     <property name="value">
      <number>50</number>
     </property>
     <property name="sliderPosition">
      <number>50</number>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
    <widget class="QDoubleSpinBox" name="MindoubleSpinBox">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>290</y>
       <width>81</width>
       <height>31</height>
      </rect>
     </property>
     <property name="wrapping">
      <bool>false</bool>
     </property>
     <property name="suffix">
      <string/>
     </property>
     <property name="decimals">
      <number>2</number>
     </property>
     <property name="maximum">
      <double>1000000.000000000000000</double>
     </property>
    </widget>
    <widget class="QLabel" name="Maxlbl">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>260</y>
       <width>51</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>MS Shell Dlg 2</family>
       <pointsize>8</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Exposure:</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_11">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>10</y>
       <width>181</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Colour (Preview)</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <zorder>lbl_LGTC_PRV</zorder>
    <zorder>lbl_LGTC_PRV_M</zorder>
    <zorder>line</zorder>
    <zorder>label_3</zorder>
    <zorder>r_out</zorder>
    <zorder>Cd_Slider_B</zorder>
    <zorder>label_4</zorder>
    <zorder>b_out</zorder>
    <zorder>bttn_multiply</zorder>
    <zorder>g_out</zorder>
    <zorder>Minlbl</zorder>
    <zorder>bttn_setExposure</zorder>
    <zorder>Cd_Slider_G</zorder>
    <zorder>label_2</zorder>
    <zorder>multSpinBox</zorder>
    <zorder>MaxdoubleSpinBox</zorder>
    <zorder>Cd_Slider_R</zorder>
    <zorder>MindoubleSpinBox</zorder>
    <zorder>Maxlbl</zorder>
    <zorder>label_11</zorder>
   </widget>
   <widget class="QWidget" name="tab_2">
    <attribute name="title">
     <string>         AOV/Target         </string>
    </attribute>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>30</y>
       <width>201</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Input name for AOV Light Group</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>50</y>
       <width>231</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="bttn_lightgroup">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>240</y>
       <width>121</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Select Target Node</string>
     </property>
    </widget>
    <widget class="QPushButton" name="bttn_removeTar">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>280</y>
       <width>231</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Remove All Targets</string>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>190</y>
       <width>161</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Select Object for Light Target</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QPushButton" name="bttn_setLG">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>80</y>
       <width>231</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Set Light Group</string>
     </property>
    </widget>
    <widget class="QPushButton" name="bttn_clearTar">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>240</y>
       <width>101</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Clear Target</string>
     </property>
    </widget>
    <widget class="QPushButton" name="bttn_LGClear">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>120</y>
       <width>231</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Clear All Light Groups</string>
     </property>
    </widget>
    <widget class="QListWidget" name="lighttar_list">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>210</y>
       <width>231</width>
       <height>21</height>
      </rect>
     </property>
    </widget>
    <widget class="Line" name="line_3">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>160</y>
       <width>231</width>
       <height>16</height>
      </rect>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </widget>
  </widget>
  <zorder>lblHeader</zorder>
  <zorder>lblOUT</zorder>
  <zorder>nodeListV</zorder>
  <zorder>node_errorOp</zorder>
  <zorder>bttn_deSelNodes</zorder>
  <zorder>bttn_nodes</zorder>
  <zorder>lcdNumber</zorder>
  <zorder>tabWidget</zorder>
 </widget>
 <resources>
  <include location="../../Light_Tool V2/GUI/main_resc.qrc"/>
 </resources>
 <connections/>
</ui>
